index.html

```diff
@@ -1,14 +1,161 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="style.css">
-  <title>Free thinking Conference: The value of reason, logic, and evidence.</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;0,900;1,300;1,900&display=swap" rel="stylesheet">
+<link rel="stylesheet" href="style.css">
+  <title>Free Thinking Conference: The value of reason, logic, and evidence.</title>
 </head>
-<body>
-  <h1>hello world!</h1>
+
+<body class="body">
+  <div class="mobile-menu hide">
+    <img class="mobile-menu-close" src="images/icons/pop-close.svg" alt="close">
+    <ul class="mobile-menu-elements">
```

Comment:
### the overall view of your design is great @danifromecuadork, but I have some comments to make it even better. 
- [ ] Please notice the page takes 100% of the view on the original design. Your page has some margin from left and right which should not be there.  `we need to use 100% of the view as visible on the original design`
- [ ] Please notice the navbar is almost pure white on the original design, as front-end engineers we are expected to mimic the original design. I suggest you do the same. 
- [ ] Besides that, please apply the appropriate font family for your text. the font that is used in the original design are `Cocogoose and Lato`
- [Optional] The active menu needs to have a color of `#EC5242` as visible on the original design.  when we are on the About` section the `About text should be `#EC5242` this helps the user to know on which page we are. 

![image](https://user-images.githubusercontent.com/69077061/226005029-8f3b2c4a-f585-45be-9dcb-d49875a5c334.png)